raven/src/main/java/com/getsentry/raven/RavenContext.java

```diff
@@ -0,0 +1,117 @@
+package com.getsentry.raven;
+
+import com.getsentry.raven.event.Breadcrumb;
+import com.getsentry.raven.util.CircularFifoQueue;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.IdentityHashMap;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * RavenContext is used to hold {@link ThreadLocal} context data (such as
+ * {@link Breadcrumb}s) so that data may be collected in different parts
+ * of an application and then sent together when (e.g.) an exception occurs.
```

Comment:
Oh, I just meant the parenthetical was confusing, because this basically expands to "then sent together when (for example) an exception occurs" which reads a little strangely.